### ISSUE TITLE
Mobile website testing through appium fails

### ISSUE THREAD

**Issue by lennyg1** [fhoeben/hsac-fitnesse-appium#7]
When testing a mobile website through appium, as soon as the browser opens, the test fails because it can not find elements. It seems I can not start any mobile website testing through Appium using hsac-fitnesse-appium.

Screenshot and page source are both empty. The browser does open on the device and working with Appium Inspector or directly via Java, I can run tests. It seems that somehow when the browser opens, the data Appium sends back to the hsac-fitnesse-appium fixture doesn't get interpreted anymore. Automating mobile apps does still work through fitnesse. It only seems to go wrong with mobile browsers.

**Comment by fhoeben** [comment]
You are right, at the moment there is no support to test mobile websites via appium. Only native apps.

Testing websites (possibly using mobile emulation) I have always done via hsac-fitnesse-fixtures' BrowserTest.

I'm afraid I'm not actively working with Appium. But PRs are always welcome of course.
